package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "node-pdfmake",
-  "version": "2.0.0",
+  "version": "3.0.0",
```

This isn't really a major update.
```suggestion
  "version": "2.0.1",
```